### ISSUE TITLE
Unable to get Dependencies during mvn clean install

### ISSUE THREAD

**Issue by DreamUniverse843** [games647/LambdaAttack#67]
It's the Terminal echo:

[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.847 s
[INFO] Finished at: 2022-09-13T15:17:22+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project lambdaattack-version-1-11: Could not resolve dependencies for project com.github.games647:lambdaattack-version-1-11:jar:2.4.0: com.github.Steveice10:MCProtocolLib:jar:1.11.2-1 was not found in https://jitpack.io during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of jitpack.io has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :lambdaattack-version-1-11

**Comment by TuxCoding** [comment]
Please post the full log and run `mvn -U`, because the version should be available.

**Comment by ryzeeepython** [comment]
Yeah, I have the same problem: Failed to read artifact descriptor for com.github.games647:lambdaattack-version-1-11:jar:2.4.0: The fot transfer artifact com.github.games647:lambdaattack-version-1-11:pom:2.4.0 from/to jitpack.io (https://jitpack.io): status code: 401, reason phrase: Unauthorized (401)
